applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -743,10 +740,10 @@ void UPwSmallStrainElement<TDim, TNumNodes>::CalculateOnIntegrationPoints(const
             Variables.StrainVector =
                 StressStrainUtilities::CalculateCauchyStrain(Variables.B, Variables.DisplacementVector);
 
-            if (rOutput[GPoint].size() != Variables.StrainVector.size())
-                rOutput[GPoint].resize(Variables.StrainVector.size(), false);
+            if (rOutput[integration_point].size() != Variables.StrainVector.size())
+                rOutput[integration_point].resize(Variables.StrainVector.size(), false);
```

Comment:
Another redundant size check:
```suggestion
            rOutput[integration_point].resize(Variables.StrainVector.size(), false);
```

Reply:
done